src/inpar/4C_inpar_scatra.cpp

```diff
@@ -166,9 +166,12 @@ void Inpar::ScaTra::set_valid_parameters(std::map<std::string, Core::IO::InputSp
                   .default_value = true}),
 
           parameter<std::string>("WRITEFLUX_IDS",
-              {.description = "Write diffusive/total flux vector fields for these scalar "
-                              "fields only (starting with 1)",
-                  .default_value = "-1"}),
+              {
+                  .description = "Write diffusive/total flux vector fields for these scalar "
+                                 "fields only (starting with 1)",
+                  .default_value = "-1",
+                  .validator = Validators::pattern(R"(^(-?\d+)(\s+-?\d+)*$)"),
```

Comment:
This is not the best fix for this parameter (one should use a vector<int>), but it allows us to test that the metadata and schema work. I will add the `input_version` next, which will also use this new validator.